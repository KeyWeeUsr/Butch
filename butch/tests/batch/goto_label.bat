echo start
echo good1
goto mylabel3

:mylabel0
echo good6
goto :eof

:mylabel1
echo good3
goto mylabel4

:mylabel2
echo good5
goto mylabel0

:mylabel3
echo good2
goto mylabel1

:mylabel4
echo good4
goto mylabel2

echo bad
