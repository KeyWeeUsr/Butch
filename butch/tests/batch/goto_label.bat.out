start
good1
good2
good3
good4
good5
good6
